aca_preview/preview.py

```diff
@@ -32,6 +32,7 @@
 PROSECO_VERSION = proseco.test(get_version=True)
 FILEDIR = Path(__file__).parent
 
+GAIA_BAD_PM = Table.read('/proj/sot/ska/analysis/gaia/agasc_gaia_xmatch_PM_gt_50mas.fits.gz')
```

Comment:
Copy this file into $SKA/data/agasc and reference accordingly (with SKA env var).  Most of time I will request a change if /proj/sot/ska appears in new code because I mostly run things on a standalone mac.